helper/tlsutil/tls.go

```diff
@@ -1,10 +1,50 @@
 package tlsutil
 
-import "crypto/tls"
+import (
+	"crypto/tls"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/vault/helper/strutil"
+)
 
 // TLSLookup maps the tls_min_version configuration to the internal value
 var TLSLookup = map[string]uint16{
 	"tls10": tls.VersionTLS10,
 	"tls11": tls.VersionTLS11,
 	"tls12": tls.VersionTLS12,
 }
+
+// ParseCiphers parse ciphersuites from the comma-separated string into recognized slice
+func ParseCiphers(cipherStr string) ([]uint16, error) {
+	suites := []uint16{}
+	ciphers := strutil.ParseDedupAndSortStrings(cipherStr, ",")
+	cipherMap := map[string]uint16{
+		"TLS_RSA_WITH_RC4_128_SHA":                tls.TLS_RSA_WITH_RC4_128_SHA,      // Grade C
```

Comment:
I'm going to merge this but remove the Grade C comments. It's not meaningful to anyone that doesn't know where those grades are coming from, and it seems odd without also adding in the grades of other ciphers, *plus* those comments can/will easily get out of date as people forget to update the grades of other ciphers over time.